src/connection/index.js

```diff
@@ -0,0 +1,12 @@
+const mysql = require("mysql2");
+
+const dbOptions = {
```

Comment:
A seperate connection file is not necessary as you already have this in your Db class.